code/__DEFINES/~skyrat_defines/jobs.dm

```diff
@@ -10,4 +10,4 @@
 #define TECH_RESTRICTED_QUIRKS "Chunky Fingers" = TRUE
 #define GUARD_RESTRICTED_QUIRKS "Blind" = TRUE, "Deaf" = TRUE, "Foreigner" = TRUE, "Pacifist" = TRUE, "Nerve Stapled" = TRUE
 
-#define FLAVOR_TEXT_CHAR_REQUIREMENT 150
+#define FLAVOR_TEXT_CHAR_REQUIREMENT 0 // Acula change: disable minimum for character flavor text (was 150)
```

Comment:
again i'd recommend making this config and PRing it to skyrat

Reply:
Well, you've seen the 2 days of trying to get a skyrat repo up and failing to do that (since GH doesn't let you have more than one fork of the same base repo, acula counts as my SR repo as far as they're concerned) so I'm stilling lacking the capacity to PR to them.

I also don't much of an interest in trying hard to sort it out at the moment, since IDK if I'll be contributing much to acula or SR anyway. (still unfortunate that it doesn't work out though)